lsst-epo/investigations-client
# [F] Add calculator questions

JIRA: https://jira.lsstcorp.org/browse/EPO-9041

## What this change does ##

Adds calculator type questions, starting with the Absolute Magnitude and Supernova Distance equations. Includes interactive (question), static (review), and XLSX variants. Some changes to the general Temml setup for better font support.

## Notes for reviewers ##

Use this branch and corresponding API. Most of the changes will be reviewed on the frontend.

Include an indication of how detailed a review you want on a 1-10 scale.
- 4

## Testing ##

- API:
    - Create a new question with the "Calculator" type and select an equation to use. Add these questions to a page.
- Client:
    - Question:
        - See that your calculator appears beneath it's text, with formatted math, serif font for the non-interactive parts, and sans serif for the inputs.
        - See that there is no value shown in the output, if no values have been input.
        - See that a value is shown if the value in the input is zero or another real number.
    - Review:
        - See that a static rendering of the equation is shown in similar format to the question 
    - XLSX:
        - See that the result of the question is added to it's table cell

## Checklist ##

If any of the following are true, please check them off

## Screenshots (optional) ##
### Before:
### After:
